cmd/initContainer/Dockerfile

```diff
@@ -1,30 +1,8 @@
-# Multi-stage docker build
-# Build stage
-FROM golang:1.17.2 AS builder
+FROM gcr.io/distroless/static
 
 LABEL maintainer="Kyverno"
 
-# LD_FLAGS is passed as argument from Makefile. It will be empty, if no argument passed
-ARG LD_FLAGS
-ARG TARGETPLATFORM
-
-ADD . /kyverno
-WORKDIR /kyverno
-
-RUN export GOOS=$(echo ${TARGETPLATFORM} | cut -d / -f1) && \
-    export GOARCH=$(echo ${TARGETPLATFORM} | cut -d / -f2)
-
-RUN go env
-
-RUN CGO_ENABLED=0 go build -o /output/kyvernopre -ldflags="${LD_FLAGS}" -v ./cmd/initContainer/
-
-# Packaging stage
-FROM scratch
-
-LABEL maintainer="Kyverno"
-
-COPY --from=builder /output/kyvernopre /
-COPY --from=builder /etc/passwd /etc/passwd
+ADD ./kyvernopre /kyvernopre
```

Comment:
Will the init container run as root? I'm testing against the local image built from this Dockerfile and get the following error:
```
  Warning  Failed     1s (x3 over 12s)  kubelet            Error: container has runAsNonRoot and image will run as root (pod: "kyverno-66957bbfd5-r47f7_kyverno(af27dd59-c5f4-4d19-bebd-16b458770433)", container: kyverno-pre)
```

Reply:
Where are you seeing this error? I was unable to replicate :(

Reply:
I was building the 1.5.2 image locally using this Makefile and saw the error after I applied definitions/install.yaml. Do you have a test image I can test with? Maybe I was doing something wrong.